packages/neos-ui/src/Containers/ErrorBoundary/index.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+export default class ErrorBoundary extends Component {
+    static propTypes = {
+        errorMessage: PropTypes.string
```

Comment:
Why is the error message coming from outside, and not from `componentDidCatch`?

Reply:
Ah, I see. So you don't want to display the actual exception to the user, right?